client/src/modules/chapters/pages/chapterPage.tsx

```diff
@@ -136,34 +151,56 @@ export const ChapterPage: NextPage = () => {
         {user &&
           (loadingChapterUser ? (
             <Spinner />
-          ) : dataChapterUser ? (
-            <HStack>
-              {dataChapterUser.chapterUser.subscribed ? (
-                <HStack>
-                  <CheckIcon />
-                  <Text>
-                    {dataChapterUser.chapterUser.chapter_role.name} of the
-                    chapter
-                  </Text>
-                  <Button onClick={() => chapterSubscribe(false)} size="md">
-                    Unsubscribe
-                  </Button>
-                </HStack>
-              ) : (
-                <Button
-                  colorScheme="blue"
-                  onClick={() => chapterSubscribe(true)}
-                  size="md"
-                >
-                  Subscribe
-                </Button>
-              )}
+          ) : (
+            dataChapterUser?.chapterUser?.subscribed && (
+              <HStack>
+                {dataChapterUser.chapterUser.subscribed ? (
+                  <HStack justifyContent={'space-between'} width={'100%'}>
+                    <Text fontWeight={500}>
+                      Unfollow upcoming chapter&apos;s events
+                    </Text>
+                    <Button onClick={() => chapterSubscribe(false)} size="md">
+                      Unsubscribe
+                    </Button>
+                  </HStack>
+                ) : (
+                  <HStack justifyContent={'space-between'} width={'100%'}>
+                    <Text fontWeight={500}>
+                      Follow upcoming chapter&apos;s events
+                    </Text>
+                    <Button
+                      colorScheme="blue"
+                      onClick={() => chapterSubscribe(true)}
+                      size="md"
+                    >
+                      Subscribe
+                    </Button>
+                  </HStack>
+                )}
+              </HStack>
+            )
```

Comment:
```suggestion
            <HStack>
              {dataChapterUser?.chapterUser?.subscribed ? (
                <HStack justifyContent={'space-between'} width={'100%'}>
                  <Text fontWeight={500}>
                    Unfollow upcoming chapter&apos;s events
                  </Text>
                  <Button onClick={() => chapterSubscribe(false)} size="md">
                    Unsubscribe
                  </Button>
                </HStack>
              ) : (
                <HStack justifyContent={'space-between'} width={'100%'}>
                  <Text fontWeight={500}>
                    Follow upcoming chapter&apos;s events
                  </Text>
                  <Button
                    colorScheme="blue"
                    onClick={() => chapterSubscribe(true)}
                    size="md"
                  >
                    Subscribe
                  </Button>
                </HStack>
              )}
            </HStack>
```

`dataChapterUser?.chapterUser?.subscribed && (` wraps the whole `HStack`, what makes impossible to reach the false of ternary operator - button to subscribe will not be displayed.

Reply:
```suggestion
            dataChapterUser?.chapterUser && (
              <HStack>
                {dataChapterUser.chapterUser.subscribed ? (
                  <HStack justifyContent={'space-between'} width={'100%'}>
                    <Text fontWeight={500}>
                      Unfollow upcoming chapter&apos;s events
                    </Text>
                    <Button onClick={() => chapterSubscribe(false)} size="md">
                      Unsubscribe
                    </Button>
                  </HStack>
                ) : (
                  <HStack justifyContent={'space-between'} width={'100%'}>
                    <Text fontWeight={500}>
                      Follow upcoming chapter&apos;s events
                    </Text>
                    <Button
                      colorScheme="blue"
                      onClick={() => chapterSubscribe(true)}
                      size="md"
                    >
                      Subscribe
                    </Button>
                  </HStack>
                )}
              </HStack>
            )
```

Sorry, just the top condition needs changing.

Reply:
No worries and thank you 💚, there is a similar condition, I will update it as well.